### ISSUE TITLE
Accessibility checks fail tests on pre-Lollipop emulators

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/android-test-kit#162]
```
What steps will reproduce the problem?
Create and run a test with accessibility checks enabled (unfortunately, I'm not 
sure how to be more specific here, it seems to affect all the tests I run). 
We've been running our tests against a Lollipop emulator and decided to try 
running them against earlier versions (tried SDKs 15, 16, & 19).

What is the expected output? What do you see instead?
Tests should pass on all SDK levels. Instead, tests pass on SDK 15 and 21, 
failing on 16 and 19.

What version of the product are you using? On what operating system?
Espresso 2.2 (running tests on Mac & Linux)

Please provide any additional information below.
Here's one of the stack traces:

java.lang.NoSuchMethodError: 
android.support.v4.view.ViewCompat.getAccessibilityLiveRegion
at 
com.google.android.apps.common.testing.accessibility.framework.ViewAccessibility
Utils.isImportantForAccessibility(ViewAccessibilityUtils.java:88)
at 
com.google.android.apps.common.testing.accessibility.framework.RedundantContentD
escViewCheck.runCheckOnViewHierarchy(RedundantContentDescViewCheck.java:56)
at 
com.google.android.apps.common.testing.accessibility.framework.integrations.espr
esso.AccessibilityValidator.runAccessibilityChecks(AccessibilityValidator.java:1
26)
at 
com.google.android.apps.common.testing.accessibility.framework.integrations.espr
esso.AccessibilityValidator.checkAndReturnResults(AccessibilityValidator.java:63
)
```

Original issue reported on code.google.com by `NHib...@gmail.com` on 1 Jul 2015 at 6:31
